cypress/integration/rich-text/RichTextPage.ts

```diff
@@ -120,6 +120,10 @@ export class RichTextPage {
       .should('be.empty')
       .as('validationErrors');
   }
+
+  expectTrackingvalue(expectedValue: any) {
+    cy.window().its('actions', { timeout: 10000 }).should('deep.equal', expectedValue);
```

Comment:
Any reason for the huge timeout? In other words, what is the difference between this assertion and existing ones e.g.  `expectNoValidationErrors`?

Reply:
Our CI is kind of slow, so I wanted to give it more time than the current `4000ms`. 

Regarding your second question, I don't see the point... `expectTrackingvalue` does something different, it accesses the `window.actions` property and asserts `expectedValue`. I did it following that link you've sent me.

Reply:
> Regarding your second question, I don't see the point.

Sorry I wasn't clear. I meant the difference in performance, not why we need it.. like why does the `expectTrackingvalue` need more time given that it's very similar to other `expectValue` & `expectNoValidationErrors` ..etc. They all access a value on the window object and assert the result. Why do we need to give the `expectTrackingvalue` more timeout?